pages/api/tags/delete-tag.js

```diff
@@ -0,0 +1,82 @@
+import * as Utilities from "~/node_common/utilities";
+import * as Data from "~/node_common/data";
+import * as ViewerManager from "~/node_common/managers/viewer";
+import * as SearchManager from "~/node_common/managers/search";
+
+export default async (req, res) => {
+  const id = Utilities.getIdFromCookie(req);
+  if (!id) {
+    return res.status(500).send({ decorator: "SERVER_GET_SLATES_TAGS", error: true });
```

Comment:
Just a heads up: for decorator naming conventions, we're moving to a system where if they're not authenticated, it's called "SERVER_NOT_AUTHENTICATED" (so that we can have a single error message for all of this same error across different endpoints)

Reply:
Got it!